### ISSUE TITLE
Regression: template system does not like MathML

### ISSUE THREAD

**Issue by dirkschulze** [tabatkins/bikeshed#1394]
I am including MathML files with the help of Bikesheds template system:

```html
<pre class=include>path: mathml/feColorMatrix01.mml</pre>
```

This results in:

```html
   <p>
    <math xmlns="http://www.w3.org/1998/Math/MathML"></math>
   </p>
   <semantics>
    <mrow>
     <mrow>
      <mfenced close="]" open="[">
```

Note the `<p>` around `<math xmlns="http://www.w3.org/1998/Math/MathML"></math>`.

The original MathML starts with:

```xml
<?xml version="1.0" encoding="UTF-8"?>

<math xmlns="http://www.w3.org/1998/Math/MathML">
 <semantics>
  <mrow>
   <mrow>
    <mfenced open="[" close="]">
     <mrow>
      <mtable>
       <mtr>
        <mtd>
         <mrow>
          <mi>R</mi>
          <mi>&apos;</mi>
         </mrow>
        </mtd>
       </mtr>
```

Maybe Bikeshed "parses" and verifies elements in templates? Anyway, any element inside the `<math>` gets pulled out of `<math>` tag.

This is a regression. When we started using it in Filter Effects, this did work.

See https://drafts.fxtf.org/filter-effects/#feColorMatrixElement

**Comment by tabatkins** [comment]
...huh. This is extremely confusing! I'm playing around with all the component parts and can't reproduce the issue, but can definitely reproduce it directly in a document. Weird!

**Comment by tabatkins** [comment]
The error even occurs if I inline the `math` into the main document, so I have *no idea* what's going on. Will continue to research.

**Comment by tabatkins** [comment]
Ah, I found the issue. Six months ago I committed <https://github.com/tabatkins/bikeshed/commit/ae68695987f5280262bf1fbb5f39523b1b3299bb>, marking `<math>` as an inline element - that is, it doesn't interrupt and auto-close `<p>` elements. (Which is true.)

This means that starting a line with a `<math>` treats the line as "text" for the Markdown parser, which means it gets wrapped in a `<p>`. If the following line is, say, an `<mrow>` tho, that is *not* an inline element, so the line gets tagged as "html"; this causes the `<p>` started from the `<math>` line to get closed, and the rest of the math contents to be output literally.

Probably what I want to do is just revert that change and treat `<math>` as a block element. All this means is that if you *do* want to put a `<math>` into a paragraph, you have to write the `<p>` yourself (or keep the `<math>` all on a single continuous line, with some text preceding it). I can't come up with a good way to keep it inline and working.

**Comment by tabatkins** [comment]
Fixed in 1aac991
